asapdiscovery-data/asapdiscovery/data/postera/cli.py

```diff
@@ -0,0 +1,179 @@
+import click
+
+from ..cli import cli
+
+
+def molecule_set_id_arg(func):
+    molecule_set_id = click.argument(
+        "molecule_set_id",
+    )
+
+    return molecule_set_id(func)
+
+
+def input_file_arg(func):
+    input_file = click.argument(
+        "input_file",
+    )
+
+    return input_file(func)
+
+
+def smiles_field_param(func):
+    smiles_field = click.option(
+        "--smiles-field",
+        help="Input csv field that stores the SMILES",
+        default="smiles",
+    )
+    return smiles_field(func)
+
+
+def id_field_param(func):
+    id_field = click.option(
+        "--id-field",
+        help="Input csv field that stores the PostEra Molecule IDs",
+        default="id",
+    )
+
+    return id_field(func)
+
+
+@cli.group(help="Commands for interacting with the PostEra API")
```

Comment:
Again would be good to mark any required args as such (if there are any. )

Reply:
Done!